src/Security/Security.php

```diff
@@ -499,7 +499,7 @@ public function ping()
     {
         HTTPCacheControlMiddleware::singleton()->disableCache();
         Requirements::clear();
-        return 1;
+        return HTTPResponse::create()->setBody('1');
```

Comment:
This isn't the right way to return a `HTTPResonse` object from a controller, really.

The response object already exists and is available through `$this->getResponse()` on any controller.